docs/book/v3/migration/v2-to-v3.md

```diff
@@ -98,3 +93,11 @@ function gravatarImage(
     string $rating = 'g'
 );
 ```
+
+#### Json
+
+The deprecated Json view helper has been removed.
+To encode data to Json for output in a view, you can call [`json_encode`](https://www.php.net/json_encode) directly.
+
+If you were relying on behaviour that was previously available via `laminas-json`, for example, calling object methods `toArray` or `toJson` prior to encoding, you should make the relevant objects implement `JsonSerializable`.
+You can find documentation on the `JsonSerializable` interface [on the PHP website](https://www.php.net/manual/en/class.jsonserializable.php).
```

Comment:
Links to the official PHP documentation should omit the language part of the URL, as the PHP documentation is available in multiple languages and redirects to the correct language automatically by using the `Accept-Language` header of the user's browser.

```suggestion
You can find documentation on the `JsonSerializable` interface [on the PHP website](https://www.php.net/manual/class.jsonserializable.php).
```

https://github.com/php/web-php/blob/c89bec8faddf40e281664555ea835857b1d1c64b/include/langchooser.inc#L5-L8